transport/conn.go

```diff
@@ -454,6 +455,16 @@ func (c *Conn) sendRequest(req frame.Request, compress, tracing bool) (frame.Res
 	return resp.Response, resp.Err
 }
 
+func (c *Conn) useKeyspace(keyspace string) error {
+	s := &Statement{Content: `USE "` + keyspace + `"`}
```

Comment:
We could have a constructor `newSatementFromCQL(col sting)` the help with that.
Also applies to cluster and control conn @Michal-Leszczynski.